packages/@o3r/design/src/core/design-token/renderers/design-token-style.renderer.ts

```diff
@@ -0,0 +1,50 @@
+import type { DesignTokenVariableSet, DesignTokenVariableStructure } from '../parsers/design-token-parser.interface';
+import { getCssTokenDefinitionRenderer } from './css/design-token-definition.renderers';
+import { getCssStyleContentUpdater } from './css/design-token-updater.renderers';
+import { existsSync, promises as fs } from 'node:fs';
+import { isAbsolute, resolve } from 'node:path';
+import type { DesignTokenRendererOptions } from './design-token.renderer.interface';
+
+/**
+ * Retrieve the function determining if a given file exists
+ * @param root Root path
+ * @param defaultFile Default file if not requested by the Token
+ */
+export const getFileToUpdateDetermination = (root = process.cwd(), defaultFile = 'styles.scss') => (token: DesignTokenVariableStructure) => {
```

Comment:
this property name is not so self-explanatory

Reply:
This is actually the purpose of commment :D.
If you see a better name, I take it ;)

Reply:
computeFileToUpdatePath?